Formula/safeint.rb

```diff
@@ -0,0 +1,56 @@
+class Safeint < Formula
+  desc "Class library for C++ that manages integer overflows"
+  homepage "https://github.com/dcleblanc/SafeInt"
+  url "https://github.com/dcleblanc/SafeInt/archive/refs/tags/3.0.28a.tar.gz"
+  version "3.0.28a"
+  sha256 "9e652d065a3cef80623287d5dc61edcf6a95ddab38a9dfeb34f155261fc9cef7"
+  license "MIT"
+  head "https://github.com/dcleblanc/SafeInt.git", branch: "master"
+
+  # The `:github_latest` strategy is used here because upstream changed their
+  # release numbering system so that 3.0.26 actually follows 3.24. Also, an
+  # optional suffix `[a-z]?` is added to capture releases like 3.0.28a.
```

Comment:
```suggestion
  # The `GithubLatest` strategy is used here because upstream changed their
  # version numbering so that 3.0.26 follows 3.24.
```

We use the class name (`GithubLatest`) when referring to a strategy (i.e., I only use the symbol name when referring to `#strategy` arguments).

At least from my perspective, we don't need to explain the added `[a-z]?`, as it's a common situation that explains itself (i.e., we only specify a regex with `GithubLatest` when we need to override the default strategy regex and this one only differs from the default in relation to the added `[a-z]?`).